.github/workflows/deploy-backend.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy Backend Docker Image
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Build the Docker image
+      working-directory: decoupled-site/backend
+      run: docker build . --file Dockerfile --tag musicfestivalimage:latest
+
+    - name: Login to ACR
+      id: ecr
+      uses: elgohr/acr-login-action@master
+      with:
+        service_principal: ${{ secrets.TF_VAR_client_id }}
+        service_principal_password: ${{ secrets.TF_VAR_client_secret }}
+        tenant: ${{ secrets.TF_VAR_tenant_id }}
+        registry: musicfestivalacr.azurecr.io
```

Comment:
In which subscription it's running? If it's on a private one, let's use findv4 subscription for all required resources.

Reply:
Currently, it's running on my personal subscription.
@jamestran1 : could you change these to findv4 subscription info in secrets?
I don't have the grant to access this repository's settings